src/index.tsx

```diff
@@ -2,7 +2,7 @@ import '@patternfly/react-core/dist/styles/base.css';
 
 import React, { useEffect, useState } from 'react';
 import { createRoot } from 'react-dom/client';
-import { RouterProvider, createBrowserRouter, createRoutesFromElements } from 'react-router-dom';
+import { RouterProvider, createBrowserRouter, createRoutesFromElements } from 'react-router';
```

Comment:
Hi @jomrazek , documentation still mentions following import:
 - `import { RouterProvider } from "react-router/dom";`

I can see you decided to go with:
 - `import { RouterProvider } from "react-router";`

Do we have more information about this? Which one should be preferred and why?

Reply:
I understood it as a way to narrow imports, given we importing more stuff I left the main package here.

Reply:
Feel free to look at https://reactrouter.com/upgrading/v6#upgrade-to-v7:~:text=Update%20DOM%2Dspecific%20imports and let me know what you think

Reply:
I know what you mean, I saw it and understood it as I explained it when I checked how the package is looking inside (not saying it is right or wrong)

Reply:
I think the doc Martin mentioned states that we should use `react-router/dom` for dom-specific imports and `react-router` for non-dom imports. RouterProvider, as a dom-specific import, should use `react-router/dom` to clearly express that we are relying on APIs optimized for the DOM environment, which aligns with the documentation's recommendations and helps reduce potential future upgrade efforts.

Reply:
Due to some additional errors, we agreed to stick with react-router for now.